dd-java-agent/instrumentation/aws-java-sns-1.0/src/main/java/datadog/trace/instrumentation/aws/v1/sns/SnsClientInstrumentation.java

```diff
@@ -0,0 +1,67 @@
+package datadog.trace.instrumentation.aws.v1.sns;
+
+import static datadog.trace.agent.tooling.bytebuddy.matcher.NameMatchers.named;
+import static net.bytebuddy.matcher.ElementMatchers.isMethod;
+
+import com.amazonaws.handlers.RequestHandler2;
+import com.google.auto.service.AutoService;
+import datadog.trace.agent.tooling.Instrumenter;
+import datadog.trace.agent.tooling.InstrumenterModule;
+import datadog.trace.bootstrap.InstrumentationContext;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import net.bytebuddy.asm.Advice;
+
+/** AWS SDK v1 SNS instrumentation */
+@AutoService(InstrumenterModule.class)
+public final class SnsClientInstrumentation extends InstrumenterModule.Tracing
+    implements Instrumenter.ForSingleType {
+  private static final String INSTRUMENTATION_NAME = "aws-sdk";
+
+  public SnsClientInstrumentation() {
+    super(INSTRUMENTATION_NAME);
```

Comment:
This instrumentation should also have its own name, so it can be disabled independently of the main one
```suggestion
    super("sns", "aws-sdk");
```